bundles/org.eclipse.core.jobs/src/org/eclipse/core/internal/jobs/JobManager.java

```diff
@@ -20,7 +20,8 @@
  *     Jan Koehnlein - Fix for bug 60964 (454698)
  *     Terry Parker - Bug 457504, Publish a job group's final status to IJobChangeListeners
  *     Xored Software Inc - Fix for bug 550738
- *     Christoph Laeubrich - remove deprecated API
+ *     Christoph Laeubrich 	- remove deprecated API
+ *      					- Issue #40 - UI freezes when a ThreadJob is waiting to run in the UI Thread
```

Comment:
Where is this documented? I find it quite usefull to not needing to query the history or bugtracker to see recent changes of a file and some basic context...

Reply:
We decided long ago in the PMC that it is ok to dd your name to the header if you want  but not adding any description as this is what Git history is for. It is somewhere in https://wiki.eclipse.org/Eclipse/PMC if you want to search for it otherwise you have to accept this in my role as PMC member. To see recent changes in the file use git blame or the history view with a filter on the resource.

Reply:
Could you add a section to https://github.com/eclipse-platform/.github/blob/main/CONTRIBUTING.md then I think it would be useful to collect all those implicit things, maybe also whether or not to change the year (I see its not always adjusted)?

Reply:
Done